mne/epochs.py

```diff
@@ -498,15 +503,44 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
             Whether to halt program execution until the figure is closed.
             Useful for rejecting bad trials on the fly by clicking on a
             sub plot.
+        n_epochs : int
+            The number of epochs per view on mne_browse_epochs. If trellis is
+            True, this parameter has no effect. Defaults to 20.
+        n_channels : int
+            The number of channels per view on mne_browse_epochs. If trellis is
+            True, this parameter has no effect. Defaults to 10.
+        fig_title : str | None
```

Comment:
The fig_title parameter changes the title of the main axis, which doesn't exist in the trellis plot. The window title could be changed, but in that case the behavior of the parameter between the two plots would be different.